dashboard/modules/serve/serve_head.py

```diff
@@ -31,16 +31,16 @@ async def get_all_deployments(self, req: Request) -> Response:
         )
 
     @routes.get("/api/serve/deployments/status")
-    @optional_utils.init_ray_and_catch_exceptions(connect_to_serve=True)
+    @optional_utils.init_ray_and_catch_exceptions(connect_to_serve=False)
     async def get_all_deployment_statuses(self, req: Request) -> Response:
-        from ray.serve.api import get_deployment_statuses
-        from ray.serve.schema import serve_application_status_to_schema
+        from ray import serve
+        from ray.serve.schema import serve_status_to_schema
 
-        serve_application_status_schema = serve_application_status_to_schema(
-            get_deployment_statuses()
-        )
+        client = serve.start(detached=True, _override_controller_namespace="serve")
+
+        serve_status_schema = serve_status_to_schema(client.get_serve_status())
         return Response(
```

Comment:
why not use `connect_to_serve` ?

Reply:
I wanted access to the client returned by `serve.start()`, so I could call `client.get_serve_status()`.

`connect_to_serve=False` also runs `serve.start()`, but I can't access the returned `client` if `serve.start()` executes in the decorator.

Reply:
serve.api._get_global_client?

Reply:
Good idea, I changed this to use `get_global_client()`.